app/Resources/views/themes/tastefulplanet/full/product_list.html.twig

```diff
@@ -0,0 +1,162 @@
+                                    {{ form_row(form.nuts) }}
+                                    {{ form_row(form.seafood) }}
+                                    {{ form_row(form.gluten) }}
+                                </div>
+                                <div class="column">
+                                    <b>{{ 'Spicyness'|trans }}: </b>
+                                    {{ form_widget(form.spicy) }}
+                                </div>
+                                <div class="column">
+                                    <b>{{ 'Servings'|trans }}: </b>
+                                    {{ form_widget(form.serving) }}
+                                </div>
+                            </div>
+                            <div class="buttons">
+                                <button class="btn btn-apply" type="submit">{{ 'Apply filters'|trans }}</button>
+                                <span class="btn btn-clear">{{ 'Clear all'|trans }}</span>
+                            </div>
+                        </div>
+                    </div>
+
+                    {{ form_end(form) }}
+                </div>
+            </div>
+
+            <div class="col-md-10 offset-md-1 product-items">
+                {% if children|length > 0 %}
+                    {% for content in children %}
+                        {{ render_esi(controller('ez_content:viewAction', {
+                            'locationId': content.contentInfo.mainLocationId,
+                            'viewType': view_type
+                        })) }}
+                    {% endfor %}
+
+                    {% if children.currentPageResults|length < children.maxPerPage %}
+                        {% set missingItems = children.maxPerPage - children.currentPageResults|length %}
+                        {% for item in 1..missingItems %}
+                            <div class="product-list-item"></div>
+                        {% endfor %}
+                    {% endif %}
+                {% else %}
+                    <div class="warning">
+                        <h2>{{ 'No results were found.'|trans }}</h2>
+                        <div class="well">
+                            <p>Search tips</p>
+                            <ul>
+                                <li>{{ 'Check spelling of keywords.'|trans }}</li>
+                                <li>{{ 'Try changing some keywords (eg, "car" instead of "cars").'|trans }}</li>
+                                <li>{{ 'Try searching with less specific keywords.'|trans }}</li>
+                                <li>{{ 'Reduce number of keywords to get more results.'|trans }}</li>
+                                <li>{{ 'Change the filters.'|trans }}</li>
+                            </ul>
+                        </div>
+                    </div>
+                {% endif %}
+            </div>
+
+            <div class="container">
+                <div class="col-lg-12 space-bottom text-center">
```

Comment:
Just a question: `col-lg-12` -> why `lg` and not `md`?

Reply:
It was like that in previous meta repo, fixed

Reply:
Keep it, we'll see later